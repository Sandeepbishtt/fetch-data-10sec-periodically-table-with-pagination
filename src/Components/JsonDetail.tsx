import React from 'react'
import {useLocation} from 'react-router-dom'

const JsonDetail =() => {
	const location = useLocation()
return (
<div>
{JSON.stringify(location.state)}
</div>
		)
}

export default JsonDetail