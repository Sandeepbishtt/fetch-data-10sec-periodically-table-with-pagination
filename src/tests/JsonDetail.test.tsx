import React from 'react';
import JsonDetail from '../Components/JsonDetail'
import {Provider} from 'react-redux'
import store from '../Redux/Store'
import ReactDOM from 'react-dom'
import {BrowserRouter} from 'react-router-dom'


describe('testing snapshot' , () => {
let container:HTMLDivElement;

beforeEach(()=>{
  container = document.createElement('div')
  document.body.appendChild(container)
  ReactDOM.render(
    <BrowserRouter>
    <Provider store={store}>
    <JsonDetail/>
  </Provider>
</BrowserRouter>,container)
})
afterEach(()=>{
  document.body.removeChild(container)
  container.remove()
})

it('render correctly',() =>{
  expect(container).toMatchSnapshot()
})

})