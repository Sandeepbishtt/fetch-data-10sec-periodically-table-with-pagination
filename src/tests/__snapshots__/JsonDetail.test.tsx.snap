// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing snapshot render correctly 1`] = `
<div>
  <div />
</div>
`;
